tests/unit/test_parser.py

```diff
@@ -1,5 +1,6 @@
 # Copyright Contributors to the Packit project.
 # SPDX-License-Identifier: MIT
+
```

Comment:
will we ever match on 0-2 empty lines after license? :D

Reply:
thanks

> there are no license headers in tests_requre/.*/__init__.py

yeah, I couldn't decide whether to remove from all `__init__.py` or add to all, so I let it as it is :-)

> will we ever match on 0-2 empty lines after license? :D

:-) should be 1 line everywhere now